recipes/pyart_mch/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "pyart_mch" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "6e16d4b947467b205f32aac9f8e309749d5f355babdae2a2467951df74e452ea" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k or win]
+  script:  # [not win]
```

Comment:
```suggestion
  script:
```

Reply:
OK. I hope it will pass now. Thanks a lot